### ISSUE TITLE
Review GitHub adminship

### ISSUE THREAD

**Issue by PeterJCLaw** [srobo/core-team-minutes#2]
From https://github.com/srobo/core-team-minutes/blob/master/2018/2018-08-14.md#action-points

**Comment by RealOrangeOne** [comment]
The [core team](https://github.com/orgs/srobo/teams/core-team) have been given access to any applicable repos. I would like to modify the default repository permissions to automatically give write access to all repos, but that's not possible without changing it for all users, which may not be what we want due to some repositories being private. There are a large number of users in the org which may no longer be active within SR, and I don't want to assume it's OK to give them access.

I've not looked into the ramifications of changing it to give everyone write access. I'll contact the trustees (primarily @jimmythompson ) to work out if they have any issues with just changing the global permissions.

**Comment by PeterJCLaw** [comment]
I don't think we need to give everyone write access to everyone, nor do I think it would be useful.
The vast majority of open source projects operate by accepting "cross-repository" PRs as their main way of contributing and I think that would work fine for us.

The alternative that I see is that we would need to go through every repo and configure branch protection (among other things) to prevent accidents from causing damage.

**Comment by RealOrangeOne** [comment]
In hindsight, I think I agree. Giving explicit access to the repos is probably easier and safer.

I do think we should protect branches anyway, and make sure tests are run against forks properly. It'll allow us to enforce PR reviews, which is a great feature for any group project.

With that in mind, i'm going to consider this fixed. With the point that it'll be handled ad-hoc as access is required / repos are added.
